internal/pkg/utils/errors/writer.go

```diff
@@ -14,8 +14,8 @@ const (
 type Writer interface {
 	Write(string)
 	WriteIndent(level int)
-	WriteBullet()
-	WritePrefix(prefix string, trace StackTrace)
+	WriteBullet(level int)
+	WritePrefix(level int, prefix string, trace StackTrace)
```

Comment:
Writer sa pouziva, ked si chces vypisat chybu nejak specialne, spravis to tak, ze typ chyby implementuje metodu `WriterError`.